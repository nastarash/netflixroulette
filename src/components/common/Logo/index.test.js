import React from 'react';
import { Logo } from './index';

describe('<Logo />', () => {
  test('should be rendered correctly', () => {
    const logo = shallow(
      <Logo />
    );
    expect(logo).toMatchSnapshot();
  });
});
