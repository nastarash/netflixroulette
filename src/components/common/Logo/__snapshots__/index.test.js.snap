// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should be rendered correctly 1`] = `
<h1
  className="logo"
>
  netflixroulette
</h1>
`;
